packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -459,3 +459,43 @@ class CenteredMessage extends StatelessWidget {
     );
   }
 }
+
+class CircularIconButton extends StatelessWidget {
```

Comment:
isn't there an existing material design close button we can use instead? this button seems a bit large and not that consistent with material design.